themes/theme-ios11/pages/Login/index.jsx

```diff
@@ -198,24 +201,32 @@ class Login extends Component {
               { /* No validate, browsers reject IDN emails! */}
               <form onSubmit={this.handleSubmitForm} noValidate className={styles.form}>
                 <TextField
+                  required
```

Comment:
i've reworked the props a bit to make the fully compatible with the TextField component.

i've removed aria-required, since it's not needed when real inputs are used with a `required` attribute, i've moved `aria-invalid` and `aria-describedby` to the `attributes` object for the `SimpleInput`.